app/players/elves/elf.py

```diff
@@ -0,0 +1,12 @@
+from abc import ABC
+from app.players.player import Player
+
+
+class Elf(Player, ABC):
```

Comment:
Player class already inherited from ABC, u don't need to do this one more time
class Elf(Player)

Reply:
But i need to recall two functions in Elf and Dwarf if i don't inherit them from ABC. Also task tell me, i shouldn't recall them there. So, what i should to do?)

Reply:
```suggestion
class Elf(Player):
```